File: snapshots/input/syntax/Main/Literals.cs
using System.Diagnostics.CodeAnalysis;

namespace Main;

[SuppressMessage("ReSharper", "all")]
public class Literals
{
    string Interpolation()
    {
        var a = 1;
        var b = 2;
        var c = 3;
        var d = 3;
        return $"a={a} b={b:0.00} c={c,24} d={d:g}";
    }
}

File: snapshots/input/syntax/VBMain/Literals.vb
Imports System.Diagnostics.CodeAnalysis

Namespace VBMain
    <SuppressMessage("ReSharper", "all")>
    Public Class Literals
        Private Function Interpolation() As String
            Dim a = 1
            Dim b = 2
            Dim c = 3
            Dim d = 3
            Return $"a={a} b={b} c={c} d={d}"
        End Function
    End Class
End Namespace
